packages/react-inbox/src/components/Inbox/index.tsx

```diff
@@ -186,20 +184,6 @@ const Inbox: React.FunctionComponent<InboxProps> = (props) => {
     [isOpen, setView, setCurrentTab]
   );
 
-  const handleBellOnMouseEnter = useCallback(
```

Comment:
i think we added the bell on hover to make make fetching the inbox faster, but since we stopped showing the "loading" state in 1.9 i don't think we need this.  

basically we were fetching 2x. one on hover and one when the inbox was open so it was duplicate